bindings/python/examples/levelset_sdf.py

```diff
@@ -0,0 +1,29 @@
+"""
+ Copyright 2023 The Manifold Authors.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ """
+
+import math
+from manifold3d import Manifold, Mesh
+
+def sphere_implicit(x, y, z):
```

Comment:
This is fine for this PR, but I'd like to move toward consistent examples. We have the same examples in our C++ samples and in our JS examples, and they're designed to be both cute and exercise the various corners of the library. I haven't made Python versions yet mostly because I'm not great at Python, but it would be nice to have the same ones to act as a bit of a Rosetta stone for our API. It would probably help me with my Python too! If you have any interest in translating, it would be much appreciated.

Reply:
Sure!   I'd already translated the gyroid one earlier to help isolate a crash on the batch function. 😄